src/rt/dwarfeh.d

```diff
@@ -473,26 +575,25 @@ ClassInfo getClassInfo(_Unwind_Exception* exceptionObject, const(ubyte)* current
 {
     ExceptionHeader* eh = ExceptionHeader.toExceptionHeader(exceptionObject);
     Throwable ehobject = eh.object;
-    //printf("start: %p '%.*s'\n", ehobject, ehobject.classinfo.info.name.length, ehobject.classinfo.info.name.ptr);
+    debug (EH_personality) writeln("start: %p '%.*s'", ehobject, cast(int)ehobject.classinfo.info.name.length, ehobject.classinfo.info.name.ptr);
     for (ExceptionHeader* ehn = eh.next; ehn; ehn = ehn.next)
     {
         // like __dmd_personality_v0, don't combine when the exceptions are from different functions
         // (fixes issue 19831, exception thrown and caught while inside finally block)
         if (currentLsd != ehn.languageSpecificData)
         {
-            // printf("break: %p %p\n", currentLsd, ehn.languageSpecificData);
+            debug (EH_personality) writeln("break: %p %p", currentLsd, ehn.languageSpecificData);
             break;
         }
 
-        //printf("ehn =   %p '%.*s'\n", ehn.object, cast(int)ehn.object.classinfo.info.name.length, ehn.object.classinfo.info.name.ptr);
+        debug (EH_personality) writeln("ehn =   %p '%.*s'", ehn.object, cast(int)ehn.object.classinfo.info.name.length, ehn.object.classinfo.info.name.ptr);
         Error e = cast(Error)ehobject;
         if (e is null || (cast(Error)ehn.object) !is null)
         {
-            currentLsd = ehn.languageSpecificData;
```

Comment:
This was a no-op - `currentLsd` is checked for being equal to `ehn.languageSpecificData` a few lines up.